me.mitya57.ReText.json

```diff
@@ -0,0 +1,378 @@
+                    "sha256": "2c3225794b40375445d1949e427ee549edfcc0a534740b3245ff70f177c40f1e"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'all:\n\ninstall:\n\tpip3 install --prefix=/app python_markdown_math-*.whl;\n' > makefile;"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "python3-markups",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://pypi.python.org/packages/56/c0/d77c6af4f43c00268372fcbaee715dc798ce70c6c6a93ae30bec32af94ed/Markups-3.0.0-py2.py3-none-any.whl",
+                    "sha256": "dd00c7f082d375bb63d6eb6873b1f984c9b474829a3ff1f9518202c64538d539"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'all:\n\ninstall:\n\tpip3 install --prefix=/app Markups-*.whl;\n' > makefile;"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "retext",
+            "no-autogen": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/2a/ba/581591a8a8d48d02236b57633ae5f55247424885e143fe1d21fba904a1ed/ReText-7.0.1.tar.gz",
+                    "sha256": "f5a2dbfb32b96423fd3e5ef00488440c36a2bcecbc6356f5c17674b265316eea"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/mquinson/po4a/archive/v0.52.tar.gz",
+                    "sha256": "547c410ad3bb8e5fd3939a6acdd39713529187be05fca7235095d57d3f0c6cfc",
+                    "strip-components": 0
+                },
+                {
+                    "type": "file",
+                    "path": "retext-man-de.po",
+                    "sha256": "9b00b30a693a3023b2f5619ab0586ddf9bd8f5bc9b75c2dabd5d6493558bb6f7"
+                },
+                {
+                    "type": "file",
+                    "path": "retext.1",
+                    "sha256": "300f04c1f808f63351b16ed0fa0f2d01b857c4d096ed71dcf54ab95281727d9a"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'disabling external renderer';",
+                        "cp 'ReText/window.py' 'ReText/window.py.bak';",
+                        "cp 'ReText/tab.py' 'ReText/tab.py.bak';",
+                        "sed -r -i 's/^([[:blank:]]*)(menuEdit\\.addAction\\(self\\.(actionWebKit|actionWebEngine)\\))([[:blank:]]*)$/\\1pass #\\2\\4/' 'ReText/window.py';",
+                        "sed -r -i 's/^([[:blank:]]*)(globalSettings\\.(useWebKit|useWebEngine)[[:blank:]]*=[[:blank:]]*)(enable)([[:blank:]]*)$/\\1\\2False\\5/' 'ReText/window.py';",
```

Comment:
I would like to avoid creating a new patch with every new release.
Moreover, it should be disabled after fixing the bug in Qt5.
https://bugreports.qt.io/browse/QTBUG-67023
https://bugreports.qt.io/browse/QTBUG-65682

Reply:
So those bugs will be fixed in Qt 5.11, right? That's due on the 31st May 2018.

Reply:
Yes, it should be fixed in Qt 5.11. However, I have already provided a workaround.

Reply:
Sure, personally I'm much more fine with a hacky sort of workaround if it's going to be removed in the next month or so than I would be if those bugs hadn't been fixed yet.